rebirthdb/__init__.py

```diff
@@ -21,11 +21,8 @@
     import builtins  # Python 3
 
 
-__all__ = ['r', 'rebirthdb'] + errors.__all__
-__version__ = version.version
-
-
-# The builtins here defends against re-importing something obscuring `object`.
```

Comment:
This comment should be moved up to line 22. We aren't obscuring `object` anymore, but the import is being used for the time being.

Reply:
You're right. moving it to line 22

Reply:
Moved